test/integration/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
```

Comment:
I see this type of files a lot. What exactly do they actually do?

Reply:
It is typically for packages that need quite a lot of intro documentation. In this case I added it because builds were failing due to "no go files in the pkg" because of the `// +integration` tag. You can read more about `doc.go` files here: https://blog.golang.org/godoc